Kickstarter.xcodeproj/project.pbxproj

```diff
@@ -7256,6 +7324,7 @@
 				A7D1F9441C850B7C000D41D5 /* Kickstarter-iOS */,
 				A755113B1C8642B3005355CF /* Library-iOS */,
 				A75511441C8642B3005355CF /* Library-iOSTests */,
+				E113BD802B7D255000D3A809 /* Library-Keychain-iOSTests */,
```

Comment:
OK, to get these tests to work, I had to make a _new testing bundle_, which is hosted, which just contains a single test, `KeychainTests`. Using an app host for `Library-iOSTests` breaks several tests, but without using the app host, `KeychainTests` will fail.

These changes look like this in the GUI:
<img width="1055" alt="Screenshot 2024-02-14 at 11 53 14 AM" src="https://github.com/kickstarter/ios-oss/assets/146007185/b66cfc1b-e461-404f-80d1-db89c49c49d3">
<img width="263" alt="Screenshot 2024-02-14 at 11 53 26 AM" src="https://github.com/kickstarter/ios-oss/assets/146007185/a467bb05-d7e8-48e6-b42a-9e0d2d5dabdf">